src/main/java/com/puppycrawl/tools/checkstyle/AstTreeStringPrinter.java

```diff
@@ -76,14 +55,14 @@ private AstTreeStringPrinter() {
     /**
      * Parse a file and print the parse tree.
      * @param file the file to print.
-     * @param withComments true to include comments to AST
+     * @param withComments {@link JavaParser.Options#WITH_COMMENTS} to include comment nodes.
```

Comment:
I know this isn't your doing, but shouldn't this parameter be named `options` and only talk about `JavaParser.Options` and not it's direct fields. If they need to know what options are available they can just look inside the enum's javadoc.

Reply:
done.